src/features/editable-table/editable-cell.tsx

```diff
@@ -0,0 +1,52 @@
+import { VariablesData } from '@/types/types';
+import { Form, Input } from 'antd';
+import { HTMLAttributes } from 'react';
+
+const { Item } = Form;
+
+interface EditableCellProps extends HTMLAttributes<HTMLElement> {
+  data: VariablesData[];
+  record: VariablesData;
+  editing: boolean;
+  dataIndex: string;
+  title: string;
+}
+
+export default function EditableCell(props: EditableCellProps) {
+  const { data, record, editing, dataIndex, title, children } = props;
+
+  return (
+    <td>
+      {editing ? (
+        <Item
+          name={dataIndex}
+          style={{ margin: 0 }}
+          rules={[
+            {
+              required: true,
+              message: `Please Input ${title}!`,
```

Comment:
```suggestion
              message: `Please Input ${title}`,
```

Please, apply `const t = useTranslations('Variables');` here and do not scream at users 😄